cmd/buildah/from.go

```diff
@@ -21,7 +21,7 @@ type fromReply struct {
 	creds           string
 	format          string
 	name            string
-	pull            bool
+	pull            string
```

Comment:
This is a breaking change.
If I previously had the script below, it will now break.